enterprise/helpers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for enterprise app.
+"""
+
+from .models import EnterpriseCustomerBrandingConfiguration
```

Comment:
@bradenmacdonald First of all, You are right about the advantage of relative imports when we rename/move the package but I think that this doesn't happen that often. The only thing I would like is that we have a common pattern of import that would contribute in our imports order and readability. If you see in other places in enterprise app then you will see absolute imports being used already, e.g. `edx-enterprise/enterprise/admin/forms.py` also in [django docs it is usually the absolute imports shown in examples](https://docs.djangoproject.com/en/1.10/intro/tutorial02/#playing-with-the-api). As long as we have consensus on whether we use [relative imports or absolute imports](http://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module) I don't have any issue :)